ui/welcome/src/androidMain/AndroidManifest.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+          package="social.androiddev.welcome" />
```

Comment:
should be `social.androiddev.ui.welcome`

Reply:
These should all be moved to gradle file (namespace). They will no longer supported in future feature of AGP. AGP 7.3.x already throws a bunch of warnings about this

Reply:
@racka98 mind doing that in your follow-up PR with the version catalog?